### PR TITLE
NVMeOf - test suite for 8GW in a single GWgroup with n-1 failures using daemon and systemctl fail method

### PR DESCRIPTION
**Pull request by HaruChebrolu**
# Description

NVMeOf - test suite for 8GW in a single GWgroup with n-1 failures in a parallel fashion using daemon fail method

polarionIds: 
[CEPH-83595555](https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83595555)
[CEPH-83595554](https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83595554)

log link: http://magna002.ceph.redhat.com/cephci-jenkins/harika/cephci-run-6DXKRV/

<summary>click to expand checklist</summary>

- [x] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [x] Review the automation design
- [x] Implement the test script and perform test runs
- [x] Submit PR for code review and approve
- [x] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Review by rahullepakshi — COMMENTED** [review]
@HaruChebrolu I am unable to see [CEPH-83595554](https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83595554) execution of systemctl stop/start in logs. Can you help me?

File: suites/squid/nvmeof/tier-2_nvmeof_1gwgroup_8gwnodes_ha_tests.yaml [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+          - tool: daemon
+            nodes:
+              - node6
+              - node7
+              - node8
+              - node9
+              - node10
+              - node11
+              - node12
+          - tool: daemon
```
Comment (rahullepakshi) [inline suites/squid/nvmeof/tier-2_nvmeof_1gwgroup_8gwnodes_ha_tests.yaml:139]: This should have been systemctl ?

Comment (HaruChebrolu) [reply suites/squid/nvmeof/tier-2_nvmeof_1gwgroup_8gwnodes_ha_tests.yaml:139]: yes. I will include it long with another fail methods. For now, lets merge with 1 fail method i.e., daemon
I will update description

**Review by rahullepakshi — APPROVED** [review]

**Review by Manohar-Murthy — APPROVED** [review]
lgtm.

**Pull request merged**